content/topics/solo-learn/java/2-control-flow/index.md

```diff
@@ -0,0 +1,18 @@
+---
+_db_id: 453
```

Comment:
Is there a reason for this id? I don't think they should be generated.

Please see: http://rocketchat.umuzi.org/group/tech-team#:~:text=Please%20don%27t%20generate%20ids%20ever